.travis.yml

```diff
@@ -1,16 +1,22 @@
 language: python
 python:
 - 2.7
-- 3.6
+- 3.5
 env:
 - TOXENV=django111
 - TOXENV=django20
+- TOXENV=django21
+- TOXENV=django22
 matrix:
   exclude:
   - python: 2.7
     env: TOXENV=django20
+  - python: 2.7
```

Comment:
Why tox env when there is no tox file.

Reply:
no idea it happening in many repos now.

Reply:
These can be removed for repos like this that don't have tox configured; it just results in the same test script running in multiple Travis workers, which is a bit wasteful.